Cargo.toml

```diff
@@ -44,28 +55,31 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 sprs = "0.11.0"
 slog = "2.7.0"
-slog-term = "2.8.1"
+slog-term = "2.9.0"
 slog-async = "2.7.0"
 smallvec = "1.8.0"
-snap = "1"
+snap = "1.0.5"
 rand = "0.8.5"
 chrono = "0.4.19"
 csv = "1.1.6"
-mimalloc = { version = "0.1.26", default-features = false }
+mimalloc = { version = "0.1.28", default-features = false }
```

Comment:
@Uzaaft : This is a dependency we have to keep at 0.1.26 for the time being because of a build error on old versions of Clang (that are used in bioconda, upon which we partially rely for distribution).  For more details on this see [this](https://github.com/microsoft/mimalloc/issues/549).  Suggest changing this back to 0.1.26 until those upstream changes are merged in.